5.x/crud-how-to.md

```diff
@@ -586,6 +586,80 @@ This is in all aspects similar to [BelongsToMany](#belongstomany) relation, the
 
 Please read the relationship [BelongsToMany](#belongstomany) documentation, everything is the same in regards to fields definition and Backpack will take care of the morphable relation saving.
 
+<a name="morphto"></a>
+#### MorphTo (n-1 relationship)
+
```

Comment:
![CleanShot 2022-08-29 at 19 17 06](https://user-images.githubusercontent.com/1032474/187247159-06c0acc6-441d-4b75-9f25-698774a5a438.png)

Reply:
We can use the screenshot above. But not here (in HOWTO)... no other relationship types have screenshots here...

But you know where we SHOULD add docs for  this too, @pxpm ? And we haven't? Inside the `relationship` field documentation 😅 Otherwise it just says that `morphTo` is not supported.

Can you please add a new section there for `morphTo`? Just with the screenshot and the basic syntax, then linking to this section in the HOWTO for detailed implementation.